app/views/templates/helpers/Breadcrumb.scala.html

```diff
@@ -0,0 +1,33 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this()
+
+@(breadcrumbTaxYear: Int)(implicit messages: Messages)
+
+<div class="govuk-breadcrumbs ">
+    <ol class="govuk-breadcrumbs__list">
+        <li class="govuk-breadcrumbs__list-item">
+            <a class="govuk-breadcrumbs__link" href="#">@messages("breadcrumb.incomeTaxAccount")</a>
+        </li>
+        <li class="govuk-breadcrumbs__list-item">
+            <a class="govuk-breadcrumbs__link" href="#">@messages("breadcrumb.incomeTaxReturn")</a>
+        </li>
+        <li class="govuk-breadcrumbs__list-item">
+            <a class="govuk-breadcrumbs__link">@messages("breadcrumb.incomeTaxYear", breadcrumbTaxYear.toString, breadcrumbTaxYear+1.toString )</a>
```

Comment:
Be careful when using +1 in situations like this, as it currently stands it is just adding 1 to the end of the value
![Screenshot 2020-10-27 at 14 21 12](https://user-images.githubusercontent.com/28566263/97314700-de91f800-185f-11eb-846a-7462e07bcbdf.png)

Reply:
I'm worried that a lot of stuff is being done that is outside of the story scope.

Reply:
changed and yh sorry think i went a bit overboard with the moving and changing of content